lib/raw.js

```diff
@@ -72,14 +72,32 @@ module.exports = function() {
       await Promise.all(_.flatMap(dat.entries, ref => {
         return _.map(['input', 'output', 'schema'], type => {
           if (ref[type]) {
-            let sname = path.basename(url.parse(ref[type]).pathname);
+            let sname = ref[type].startsWith('http://') ?
+              path.basename(url.parse(ref[type]).pathname) :
+              ref[type].replace(/#$/, '');
             // This JSON.parse stuff here and a few lines down lets us avoid going remote for remote refs
             // that are actually local. We do not allow remote (as in cross-project) refs in taskcluster.
             let schema = JSON.parse(fs.readFileSync(path.join('raw/reference', project, 'schemas', sname)));
```

Comment:
Something to think about for the future is to just load all of the schemas and then index them by their $id and load them that way.  Let's not do that yet, though :)